libs/ui-watt/src/lib/foundations/_variables.scss

```diff
@@ -81,7 +81,7 @@
   --watt-typography-headline-color: var(--watt-color-primary-dark);
   --watt-typography-text-color: var(--watt-on-light-high-emphasis);
   --watt-typography-button-color: var(--watt-color-primary);
-  --watt-typography-label-color: var(--watt-color-primary-dark);
+  --watt-typography-label-color: var(--watt-color-neutral-black);
```

Comment:
question: should it not be `--watt-on-light-high-emphasis`?

![image](https://user-images.githubusercontent.com/1096332/210780726-00c60cb5-efd7-449e-916f-8cd35f12e624.png)

Reply:
yes it should 🤦‍♂️

Reply:
Oh, sorry for being annoying but just noticed that the value of `--watt-on-light-high-emphasis` has changed. Can you adjust it too?

Reply:
on it 😎